packages/react-components/lib/robots/robot-info.tsx

```diff
@@ -20,8 +20,7 @@ const useStyles = makeStyles((theme) =>
   createStyles({
     root: {
       '&$disabled': {
-        color: theme.fontColors,
-        borderColor: theme.fontColors,
+        borderColor: theme.palette.divider,
```

Comment:
I think the default border color is not the same as the divider color, so doing this would mess up the default material theming.